js/offcanvas.js

```diff
@@ -36,14 +36,14 @@
     if (this.options.autohide && !this.options.modal) {
       var eventName = (navigator.userAgent.match(/(iPad|iPhone)/i) === null) ? 'click' : 'touchstart'
       $(document).on('click touchstart', $.proxy(this.autohide, this))
-    }
-
-    if (this.options.toggle) this.toggle()
+    }   
 
-    if (this.options.disablescrolling) {
+    if (typeof(this.options.disablescrolling) === "boolean") {
```

Comment:
If you use `data-disablescrolling="false"`, the option will be a string and not a boolean. This doesn't seem to fix the issue.

This also breaks setting `"disablescrolling": false` in JavaScript. `false` is a boolean, so scrolling will be disabled, but that's as intended.